(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[233],{5615:function(e,s,i){"use strict";i.r(s),i.d(s,{default:function(){return h}});var n=i(9008),r=i(972),a=i(2339),t=i.n(a),o=i(9369),c=i(7294),l=i(5893);function h(){var e=(0,c.useContext)(o.N).state.darkMode;return(0,l.jsxs)(r.Z,{children:[(0,l.jsxs)(n.default,{children:[(0,l.jsx)("title",{children:"Help | PHD"}),(0,l.jsx)("link",{rel:"shortcut icon",href:"favicon.ico"})]}),(0,l.jsxs)("div",{className:t().wrapper,children:[(0,l.jsx)("h1",{children:"About"}),(0,l.jsxs)("p",{children:["PHD (",(0,l.jsx)("u",{children:"P"}),"hage & ",(0,l.jsx)("u",{children:"H"}),"ost ",(0,l.jsx)("u",{children:"D"}),"aily) is a web application that combines information on phage-host interactions from seven sources \u2014"," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/",children:"NCBI Virus"}),","," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.genome.jp/virushostdb/",children:"Virus-Host DB"}),","," ",(0,l.jsx)("a",{target:"_blank",href:"https://mvp.medgenius.info/home",children:"MVP"}),","," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.ncbi.nlm.nih.gov/refseq/",children:"RefSeq"}),","," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.ncbi.nlm.nih.gov/genbank/",children:"GenBank"}),","," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.uniprot.org/",children:"UniProt"}),", and"," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.ebi.ac.uk/intact/home",children:"IntAct"}),". PHD provides information on hosts for prokaryotic viruses at the species resolution using two alternative taxonomic classification systems,"," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.ncbi.nlm.nih.gov/taxonomy",children:"NCBI Taxonomy"})," ","or"," ",(0,l.jsx)("a",{target:"_blank",href:"https://gtdb.ecogenomic.org/",children:"Genome Taxonomy Database (GTDB)"}),". Analogously, virus species are classified according to"," ",(0,l.jsx)("a",{target:"_blank",href:"https://talk.ictvonline.org/",children:"NCBI Taxonomy or International Committee on Taxonomy of Viruses (ICTV)"}),"."]}),(0,l.jsxs)("p",{children:["PHD also points to genome assemblies available for each virus species by keeping track of the"," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.ncbi.nlm.nih.gov/assembly",children:"NCBI Assembly"})," ","resource and the"," ",(0,l.jsx)("a",{target:"_blank",href:"https://github.com/RyanCook94/inphared",children:"INPHARED"})," ","database of complete phage genomes."]}),(0,l.jsx)("p",{children:"PHD also publishes daily reports on the current catalog of phage-host interactions."}),(0,l.jsx)("h1",{children:"Methods"}),(0,l.jsx)("p",{children:"PHD integrates information from several data sources to collect data related to virus genomic sequences, host information, and taxonomic classification."}),(0,l.jsx)("img",{src:e?"method-dark.svg":"method-light.svg",alt:"phd workflow"}),(0,l.jsxs)("ol",{className:t().points,children:[(0,l.jsxs)("li",{children:["Names and/or NCBI taxonomy identifiers (",(0,l.jsx)("span",{children:"taxIds"}),") of hosts are extracted from nucleotide/protein sequence records of viruses available in six source databases:",(0,l.jsxs)("ul",{className:t().dots,children:[(0,l.jsx)("li",{children:"NCBI Virus (from XML file)"}),(0,l.jsxs)("li",{children:["RefSeq (from"," ",(0,l.jsx)("a",{target:"_blank",href:"https://ftp.ncbi.nlm.nih.gov/refseq/release/viral/",children:"GenBank flat"})," ","files under the qualifiers ",(0,l.jsx)("span",{children:"/host="})," and"," ",(0,l.jsx)("span",{children:"/lab_host="}),")"]}),(0,l.jsxs)("li",{children:["Virus-Host DB (from"," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.genome.jp/ftp/db/virushostdb/virushostdb.tsv",children:"TSV file"}),")"]}),(0,l.jsxs)("li",{children:["MVP (from"," ",(0,l.jsx)("a",{target:"_blank",href:"https://mvp.medgenius.info/Downloads/",children:"TSV files"}),")"]}),(0,l.jsxs)("li",{children:["UniProt-SwissProt (from"," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.uniprot.org/uniprot/?query=taxonomy:%22Viruses%20[10239]%22&format=tab&columns=id,organism,organism-id,virus%20hosts&sort=score&fil=reviewed:yes",children:"protein sequence entries in UniProt-SwissProt"}),")"]}),(0,l.jsxs)("li",{children:["IntAct (from protein-protein interactions available on the"," ",(0,l.jsx)("a",{target:"_blank",href:"ftp://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/",children:"IntAct FTP server"}),")"]})]})]}),(0,l.jsxs)("li",{children:["The extracted host names/",(0,l.jsx)("span",{children:"taxIds"})," are queried in"," ",(0,l.jsx)("a",{target:"_blank",href:"https://github.com/shenwei356/taxonkit",children:"TaxonKit"})," ","against NCBI Taxonomy to retrieve full taxonomic lineages of hosts including their names, ranks, and taxIds. Only prokaryotic host species (from Bacteria or Archaea) are included in further steps. For example:",(0,l.jsx)("pre",{children:"Escherichia coli species 562 Escherichia genus 561 Enterobacteriaceae family 543 Enterobacterales order 91347 Gammaproteobacteria class 1236 Proteobactertia phylum 1224 Bacteria superkingdom 2"})]}),(0,l.jsxs)("li",{children:["An additional taxonomic information (if available) for each prokaryotic host species is retrieved from"," ",(0,l.jsx)("a",{target:"_blank",href:"https://gtdb.ecogenomic.org/",children:"Genome Taxonomy Database (GTDB)"}),"."]}),(0,l.jsx)("li",{children:"Interaction assignments between virus sequence records and the prokaryotic host species are collected from the source databases."}),(0,l.jsxs)("li",{children:["Virus ",(0,l.jsx)("span",{children:"taxIds"})," provided in sequence records are used to retrieve virus taxonomic lineages from NCBI Taxonomy. The obtained virus species ",(0,l.jsx)("span",{children:"taxIds"})," or sequence accessions are used to retrieve virus taxonomic lineages (if available) in"," ",(0,l.jsx)("a",{target:"_blank",href:"https://talk.ictvonline.org/taxonomy/vmr/",children:"International Committee on Taxonomy of Viruses (ICTV)"}),". Sequence accessions are then assigned to the appropriate virus species. For example, three genomic sequences (",(0,l.jsx)("span",{children:"MN125599"}),", ",(0,l.jsx)("span",{children:"MN125600"}),", and ",(0,l.jsx)("span",{children:"NC_049813"}),") belong to the"," ",(0,l.jsx)("i",{children:"Escherichia"})," virus 12210I species."]}),(0,l.jsxs)("li",{children:["Sequence accessions within virus species are grouped into genome assemblies based on metadata provided in the NCBI Assembly database. For example, two sequence accessions - ",(0,l.jsx)("span",{children:"MN125599"}),","," ",(0,l.jsx)("span",{children:"MN125600"})," - are part of one genome assembly from GenBank (assembly accession: ",(0,l.jsx)("span",{children:"GCA_009903655"}),") while the third sequence ",(0,l.jsx)("span",{children:"NC_049813"})," is a separate genome assemble from RefSeq (assembly accession: GCF_009671745). Assembly level category (i.e., ",(0,l.jsx)("i",{children:"Complete"})," or ",(0,l.jsx)("i",{children:"Scaffold"})," or ",(0,l.jsx)("i",{children:"Contig"})," or"," ",(0,l.jsx)("i",{children:"unknown"}),") is assigned to each virus assembly based on information provided by"," ",(0,l.jsx)("a",{target:"_blank",href:"https://www.ncbi.nlm.nih.gov/assembly",children:"NCBI Assembly"})," ","and"," ",(0,l.jsx)("a",{target:"_blank",href:"https://github.com/RyanCook94/inphared",children:"INPHARED"})," ","databases."," "]}),(0,l.jsxs)("li",{children:["Source databases are assigned to each interaction between virus and host species. For example, the interaction between"," ",(0,l.jsx)("i",{children:"Escherichia"})," virus 12210I and ",(0,l.jsx)("i",{children:"E. coli"})," was covered by three source databases (i.e., NCBI Virus, Virus-Host DB, and RefSeq)."]})]}),(0,l.jsx)("h1",{children:"Cite PHD"}),(0,l.jsx)("p",{children:"If you find PHD useful in your work please cite:"}),(0,l.jsx)("i",{children:"Albrycht K et al. (2022). Daily reports on phage-host interactions."})]})]})}},4156:function(e,s,i){(window.__NEXT_P=window.__NEXT_P||[]).push(["/help",function(){return i(5615)}])},2339:function(e){e.exports={wrapper:"help_wrapper__je-oY",dots:"help_dots__mdOzW",points:"help_points__1QiZw"}}},function(e){e.O(0,[523,773,972,774,888,179],(function(){return s=4156,e(e.s=s);var s}));var s=e.O();_N_E=s}]);